src/routes/settings-access-status-types-route/settings-access-status-types-route.js

```diff
@@ -48,38 +49,33 @@ const SettingsAccessStatusTypesRoute = ({
     getAccessTypes(params.kbId);
```

Comment:
For some reason sonar shows that both `useEffect` are covered, but there are no tests. Please add two tests: to check that `getAccessTypes` is called only once when component is mounted and to check that `getAccessTypes` is called only when `params.kbId` changes